src/main/java/edu/ie3/datamodel/io/csv/BufferedCsvWriter.java

```diff
@@ -106,6 +106,11 @@ private void writeFileHeader(String[] headLineElements) throws IOException {
   private void writeOneLine(String[] entries) throws IOException {
     for (int i = 0; i < entries.length; i++) {
       String attribute = entries[i];
+      if (attribute.matches("^\"\\{(?:.*)\\}\"$")) {
+        attribute = attribute.replaceAll("\"", "\"\"");
+        attribute = attribute.substring(1, attribute.length() - 1);
+      }
```

Comment:
Please make this a distinct method, that later can be moved to `StringUtils` in PowerSystemUtils. This method has the following tasks:

1. Identify, if the String has to be quoted: It contains the csv separator character, the character " or any other special character (\n, ...)
2. Replace all single " by ""
3. Add quotation marks at the beginning and the end of the string (you can reuse the Regex in `StringUtils#quote(String)`)

Additionally, please code an equivalent method, that does revert that steps.